### PR TITLE
change dashboard css to less and tweak colors

### PR DESCRIPTION
**Pull request by czue**
https://github.com/dimagi/commcare-hq/pull/26406#discussion_r366922486

My plan is to merge this and deploy to india and then revert if there are any issues.

### CODE REVIEW

File: custom/icds_reports/templates/icds_reports/dashboard.html [hunk 1]
```diff
@@ -16,7 +16,8 @@
         <link type="text/css" rel="stylesheet" media="all" href="{% static 'leaflet/dist/leaflet.css' %}">
         <link type="text/css" rel="stylesheet" media="all" href="{% static 'angular-busy/dist/angular-busy.min.css' %}"/>
         <link type="text/css" rel="stylesheet" media="all" href="{% static 'perfect-scrollbar/css/perfect-scrollbar.min.css' %}"/>
-        <link type="text/css" rel="stylesheet" media="all" href="{% static 'css/icds_dashboard.css' %}"/>
+        <link type="text/less" rel="stylesheet" media="all" href="{% static 'css/icds_dashboard.less' %}"/>
+        <link type="text/less" rel="stylesheet" media="all" href="{% static 'css/fact_sheets.less' %}"/>
```
Comment (czue) [inline custom/icds_reports/templates/icds_reports/dashboard.html]: was thinking that we could opportunistically start refactoring styles into smaller files as we make changes, though also open to the idea that this makes things worse without some initial effort at organization (if we decide the latter I'll just move these changes into `icds_dashboard.less` for now).

File: custom/icds_reports/templates/icds_reports/dashboard.html [hunk 2]
```diff
@@ -104,6 +105,9 @@
         <script src="{% static 'js/directives/access-denied/access-denied.directive.js' %}"></script>
         <script src="{% static 'js/directives/info-message/info-message.directive.js' %}"></script>
 {% endcompress %}
+{% if less_debug %}
+  <script src="{% static 'less/dist/less.min.js' %}"></script>
+{% endif %}
```
Comment (czue) [inline custom/icds_reports/templates/icds_reports/dashboard.html]: (this was what tripped me up locally)

File: custom/icds_reports/templates/icds_reports/dashboard.html (hunk 1 above)
Comment (orangejenny) [reply custom/icds_reports/templates/icds_reports/dashboard.html]: I'm in favor of this - I'm definitely glad to see fewer inline styles, and also generally like having more small, well-named files.

**Review by orangejenny — APPROVED** [review]

**Comment by czue** [comment]
deployed india and all looks clear

**Pull request merged**